actions/shared/create_cluster/action.yml

```diff
@@ -1,18 +1,25 @@
-name: 'Custom GitHub Action'
-description: 'A GitHub Action that runs kind'
-#inputs:
+name: 'Create Kubernetes cluster Action'
+description: 'A GitHub Action that creates Kubernetes cluster using Kind'
+inputs:
+  pipeline_branch:
+    description: 'branch in qubership-test-pipelines repository'
+    required: true
+    default: 'main'
```

Comment:
I guess individual actions should not have `pipeline_branch` input

Reply:
pipeline_branch deleted